app/controllers/templates_controller.rb

```diff
@@ -95,6 +95,10 @@ def preview
     safe_render(@template)
   end
 
+  def export
+    send_data @template.to_erb, :type => 'text/plain', :disposition => 'attachment', :filename => @template.filename
```

Comment:
Sorry, please disregard, I meant to put it against the API controllers. I'll log a different comment there though.